graphite/graphite.go

```diff
@@ -100,6 +101,15 @@ func readTraceback(resp *http.Response) (*[]string, error) {
 // DefaultClient is the default HTTP client for requests.
 var DefaultClient = &http.Client{
 	Timeout: time.Minute,
+	Transport: &http.Transport{
+		MaxIdleConnsPerHost: 0,
```

Comment:
I think that's a fine plan. We've tried many things to fix these errors, and that would make it pretty easy. I'm worried about performance, though, since we do a lot of connections between bosun and tsdb.